src/utils.js

```diff
@@ -238,3 +238,21 @@ export function buildUrl(location, values) {
 
   return isEmpty(params) ? url : `${url}?${queryString.stringify(params)}`;
 }
+
+export function formatNoteReferrerEntityData(data) {
+  if (data) {
```

Comment:
I would suggest checking firstly for `!data` so we don't have the code indented when `data` is truthy.